### PR TITLE
Update changelog for v0.5.0 beta

### PR DESCRIPTION
**Pull request by jchengsfx**
First draft of changelog. Mainly just copied the commit messages, so the wording and categorization may need to be adjusted by someone more knowledgable.  Also, copied the Traces, Metrics, and Extensions sections from v0.4.0 (not sure what should be added/removed).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,79 @@
 
 ## Unreleased
 
+## v0.5.0 Beta
+
+## 🛑 Breaking changes 🛑
+
+- Enable "new-metrics" by default (#1148)
```
Comment (tigrannajaryan) [inline CHANGELOG.md:9]:
```suggestion
- Make "--new-metrics" command line flag the default (#1148)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,79 @@
...
+- Remove `export_resource_labels` flag from Zipkin exporter (#1163)
+- Add config settings for Http Client, change Zipkin to use it. (#1186)
```
Comment (tigrannajaryan) [inline CHANGELOG.md:11]: This needs to tell what is changed in Zipkin config. If you check the PR testdata.yaml shows it: "url" is renamed to "endpoint".

>Add config settings for Http Client
This is not worth mentioning, it is an internal change.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,79 @@
...
+- Update OTLP to v0.4.0 (#1142)
```
Comment (tigrannajaryan) [inline CHANGELOG.md:12]: This is major breaking change. It makes Collector incompatible with any other sender or receiver of OTLP protocol of different version. This needs big red warning.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,79 @@
 
 ## Unreleased
 
+## v0.5.0 Beta
+
+## 🛑 Breaking changes 🛑
+
+- Enable "new-metrics" by default (#1148)
+- Remove `export_resource_labels` flag from Zipkin exporter (#1163)
```
Comment (tigrannajaryan) [inline CHANGELOG.md:10]: This is a minor breaking change.

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,79 @@
...
+- Add config settings for Http Client, change Zipkin to use it. (#1186)
+- Update OTLP to v0.4.0 (#1142)
+
+## ⚠️ Major changes ⚠️
+
+- Split OTLP receiver by protocols, allow mTLS support (#1223)
```
Comment (tigrannajaryan) [inline CHANGELOG.md:16]: This is a breaking change too. It changes config file format.

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,79 @@
...
+- Use tls_settings consistently between gRPC and HTTP (#1233)
```
Comment (tigrannajaryan) [inline CHANGELOG.md:17]: This is a breaking config change.

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,6 +2,79 @@
...
+
+## 🚀 New components 🚀
+
+- Receivers
+  - Added process scraper to the `hostmetrics` receiver (#1047)
+
+## 💡 Enhancements 💡
+
+- otlpexporter: send configured headers in request (#1130)
+- Add protocol server settings and remove endpoint from ReceiverSettings (#1172)
```
Comment (tigrannajaryan) [inline CHANGELOG.md:27]: Internal change, not worth changelog.

File: CHANGELOG.md [hunk 8]
```diff
@@ -2,6 +2,79 @@
...
+- Add a NetAddr that can be used for HTTP or other protocols (#1188)
```
Comment (tigrannajaryan) [inline CHANGELOG.md:28]: Internal change, not worth changelog.

File: CHANGELOG.md [hunk 9]
```diff
@@ -2,6 +2,79 @@
...
+- Enable Collector to be run as a Windows service (#1120)
+- Add config for HttpServer (#1196)
+- Allow cors in HTTPServerSettings (#1211)
+- Add a generic grpc server settings config, cleanup client config (#1183)
+- Rely on gRPC to batch and loadbalance between connections instead of custom logic (#1212)
+- Allow to tune the read/write buffers for gRPC clients (#1213)
+- Allow to tune the read/write buffers for gRPC server (#1218)
+- Bump jaeger version to v1.8.2 with thrift 0.13 (#1203)
```
Comment (tigrannajaryan) [inline CHANGELOG.md:36]: Internal change, not worth changelog.

File: CHANGELOG.md [hunk 10]
```diff
@@ -2,6 +2,79 @@
...
+| OpenCensus | Filter | Logging |
+| OTLP | Memory Limiter | OpenCensus |
+| Prometheus || OTLP |
+| VM Metrics || Prometheus |
+
+### Extensions
+
+- Health Check
+- Performance Profiler
+- zPages
```
Comment (tigrannajaryan) [inline CHANGELOG.md:76]: Not sure why we need this.

File: CHANGELOG.md [hunk 11]
```diff
@@ -2,6 +2,44 @@
...
+
+## 💡 Enhancements 💡
+
+- otlpexporter: send configured headers in request (#1130)
+- Enable Collector to be run as a Windows service (#1120)
+- Add config for HttpServer (#1196)
+- Allow cors in HTTPServerSettings (#1211)
+- Add a generic grpc server settings config, cleanup client config (#1183)
+- Rely on gRPC to batch and loadbalance between connections instead of custom logic (#1212)
+- Allow to tune the read/write buffers for gRPC clients (#1213)
```
Comment (bogdandrutu) [inline CHANGELOG.md:30]: Defaults for these two #1213 and #1218 improved performance by ~20%

**Pull request merged**